cross-functional/workflow.md

```diff
@@ -1,34 +1,125 @@
+
+````{admonition} Example structure of an initiative
+:class: dropdown
+Here's the rough structure of an initiative, use this to guide the creation of new initiatives.
+
+```markdown
+title: Initiative name in GitHub issue
+
+### Description
+
+A short description of the initiative, with enough context so that people can understand
+what kind of work it will entail, and why it is important to 2i2c's overall goals.
+
+### Linked goal or project
+
+- Link to a strategic goal or a project (2i2c-based, upstream, etc)
+
+### Why is this important?
+
+- This will allow us to XXX
+
+### Measures and definition of done
 
-The [`#solution-forum` Slack channel](https://2i2c.slack.com/archives/C06G5FAAT63) is a space for cross-organization initiatives and strategic conversation.
+- We expect to see XXX
+- This will be complete when YYY
 
-- It is meant for discussing organization-wide practices, policies, and planning.
-- It will have a combination of strategic- and tactical-level discussion. (in the future we may split this into two channels if it's too much)
-- It will initially focus its discussion around our "cross-functional initiatives" board and the items in it.
-- Anyone that participates in conversations in this channel should dedicate the attention to understand the context for discussion and participate until topics are resolved.
-- Area leads are expected to monitor this channel and participate in cross-team conversations as they arise.
-- All 2i2c staff have visibility into the channel and may participate in conversations, but are not expected to do so as part of their daily responsibilities.
+```[tasklist]
+- [ ] A growing list of tasks and actions used to coordinate work around this initiative.
+- [ ] Each should be an actionable issue that we track independently.
+- [ ] This may be empty when the initiative is created! The initiative's goals should be used to decide what action issues to create next.
+```
+````
 
-### Team members expected to monitor this channel
+### Bi-weekly initiatives meetings
 
-Below is a list of the team members that we **expect** to participate in this channel on a regular basis:
+There are two bi-weekly meetings with the initiatives team.
 
+- **Initiative planning and refinement**: A combindation of **coordination** and **prioritization**. We first walk the board and ensure each initiative has clear next steps, assigned people, and that any coordination between teams is clear. We next zoom out and discuss our overall portfolio of initiatives to ensure we're sequencing them properly.
+- **Initiative retrospectives**: The Initiatives team follows the same retrospective structure we follow for any team.
+
+Our goal is to have the **Initiatives planning meeting** before any other sprint planning meetings, and to have the **Initiatives retrospective** after any other team retrospectives.
```

Comment:
I agree with the intent here, but I think James caught something important - before/after doesn't matter as much as having Initiatives wrap around day-to-day work, and day-to-day work be aligned to bigger picture initiatives.  

So maybe we say "Our goal is to have the Initiatives planning meeting inform priorities for other sprint planning, and the Initiatives retrospective include actual results from sprint delivery."

Reply:
I like @aprilmj's framing, will use her language. I agree it's more important to state the goal in general rather than define a specific policy around sequencing...we can figure out the specifics as we iterate